### ISSUE TITLE
Backup or Restore not working with Raspbian

### ISSUE THREAD

**Issue by oldjiberjaber** [rlancaste/AstroPi3#13]
The "Backup or Restore" does not work under Raspbian.  The Desktop Entry/Command is set to execute mate-terminal and this fails on Raspbian.  Workaround is to replace with "/home/pi/AstroPi3/backupOrRestore.sh" and ensure Execute in terminal emulator is ticked.

Drag and drop of directory does not work in this terminal approach however.

**Comment by rlancaste** [comment]
Yes, the script was originally developed for Ubuntu-MATE.  I added Raspbian a few months ago.  This might be good to include as well.
